src/index.js

```diff
@@ -291,6 +291,14 @@ const indy = {
     return IndySdk.deleteWallet(JSON.stringify(config), JSON.stringify(credentials))
   },
 
+  exportWallet(wh: WalletHandle, exportConfig: Object): Promise<void> {
+    return IndySdk.exportWallet(wh, JSON.stringify(exportConfig))
```

Comment:
Could you please throw an Error for the iOS platform? The same for the second method.

```suggestion
    if (Platform.OS === 'ios') {
      throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
    }

    return IndySdk.exportWallet(wh, JSON.stringify(exportConfig))
```

Reply:
@jakubkoci It is already implemented for iOS.

@TimovanDijk can't test on iOS, so I'll test it on iOS and report back